platform/lib/domain/src/main/kotlin/io/hamal/lib/domain/vo/Namespace.kt

```diff
@@ -19,4 +22,17 @@ class NamespaceName(override val value: String) : ValueObjectString() {
     companion object {
         val default = NamespaceName("default")
     }
-}
\ No newline at end of file
+}
+
+class NamespaceFeatures(override var value: HotObject = HotObject.empty) : ValueObjectHotObject() {
```

Comment:
It would be great to have a function + unit tests for something like

fun hasFeature(feature: Feature) : Boolean = ....

false or absent --> false
found and true -> true

Reply:
Did not thought about putting it as a HotObject but I think thats a cool idea